Sources/TuistLoader/Mappers/ProjectDescriptionHelpersLocationMapper.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  ProjectDescriptionHelpersLocationMapper.swift
+//  TuistLoader
+//
+//  Created by Luis on 11/10/20.
+//
+
+import Foundation
+import TSCBasic
+import TuistCore
+import TuistSupport
+
+/// Mapper that changes the development region based on configuration
+/// in the Config.swift
+public final class ProjectDescriptionHelpersLocationMapper: ProjectMapping {
+    private let config: TuistCore.Config
+    private let rootDirectoryLocator: RootDirectoryLocating
+
+    private lazy var derivedHelpersDirectory: AbsolutePath? = {
+        guard let configPath = config.path,
+              let rootDirectory = rootDirectoryLocator.locate(from: configPath) else {
+            return nil
+        }
+
+        return rootDirectory
+            .appending(component: Constants.tuistDirectoryName)
+            .appending(component: Constants.helpersDirectoryName)
+            .appending(component: Constants.DerivedDirectory.name)
+    }()
+
+    public init(
+        config: TuistCore.Config,
+        rootDirectoryLocator: RootDirectoryLocating = RootDirectoryLocator()
+    ) {
+        self.config = config
+        self.rootDirectoryLocator = rootDirectoryLocator
+    }
+
+    // MARK: - ProjectMapping
+
+    public func map(project: TuistCore.Project) throws -> (TuistCore.Project, [SideEffectDescriptor]) {
```

Comment:
Do we need to prepend `Project` with `TuistCore`? 🤔  (this appears to be in multiple places here)

Reply:
Not needed, removing